qemu/tests/nic_hotplug.py

```diff
@@ -109,6 +113,23 @@ def device_add_nic(pci_model, netdev, device_id):
         add_output = vm.monitor.send_args_cmd(pci_add_cmd)
         return add_output
 
+    def reboot_vm(vm, need_session=True, serial=True):
```

Comment:
is there any chance to use need_session=True? i saw you called reboot_vm twice in this file without need_session parameter. so what scenario will you use this parameter?

Reply:
yes , xiywang , I planed to pass "False" to the last time reboot.  updated. thanks